### PR TITLE
Update Sixth Pillar feats to Ruby Phoenix reprint

### PR DESCRIPTION
**Pull request by mysurvive**
Updating descriptions on Sixth Pillar feats.

### CODE REVIEW

File: packs/data/feats.db/six-pillars-stance.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         },
         "rules": [],
         "source": {
-            "value": "Pathfinder #167: Ready? Fight!"
```
Comment (simonward) [inline packs/data/feats.db/six-pillars-stance.json:28]: If there are changes in a newer release, it should generally take precedence.
The updated source should be: "Pathfinder Fists of the Ruby Phoenix Hardcover Compilation"

File: packs/data/feats.db/maneuvering-spell.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
         },
         "rules": [],
         "source": {
-            "value": "Pathfinder #167: Ready? Fight!"
```
Comment (simonward) [inline packs/data/feats.db/maneuvering-spell.json:28]: If there are no changes, the old source is typically left alone.

File: packs/data/feats.db/six-pillars-stance.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
...
+            "value": "Fists of the Ruby Phoenix Hardcover, Pathfinder #167: Ready? Fight!"
```
Comment (simonward) [inline packs/data/feats.db/six-pillars-stance.json:28]: If there are changes in a newer release, it should generally take precedence.
The updated source should be: "Pathfinder Fists of the Ruby Phoenix Hardcover Compilation"

File: packs/data/feats.db/maneuvering-spell.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
...
+            "value": "Fists of the Ruby Phoenix Hardcover, Pathfinder #167: Ready? Fight!"
```
Comment (simonward) [inline packs/data/feats.db/maneuvering-spell.json:28]: If there are no changes, the source is typically left alone.

File: packs/data/feats.db/six-pillars-stance.json (hunk 3 above)
Comment (mysurvive) [reply packs/data/feats.db/six-pillars-stance.json]: fixed

File: packs/data/feats.db/maneuvering-spell.json (hunk 4 above)
Comment (mysurvive) [reply packs/data/feats.db/maneuvering-spell.json]: Thanks, fixed

File: packs/data/feats.db/touch-focus.json [hunk 5]
```diff
@@ -38,11 +39,12 @@
                 "key": "RollOption",
                 "label": "Use Touch Focus",
```
Comment (TikaelSol) [inline packs/data/feats.db/touch-focus.json:40]:
```suggestion
```

File: packs/data/feats.db/touch-focus.json [hunk 6]
```diff
@@ -38,11 +39,12 @@
...
                 "option": "touch-focus",
+                "slug": "touch-focus-option",
```
Comment (TikaelSol) [inline packs/data/feats.db/touch-focus.json:42]:
```suggestion
```

File: packs/data/feats.db/touch-focus.json [hunk 7]
```diff
@@ -30,6 +30,7 @@
                     "touch-focus"
                 ],
                 "selector": "spell-attack",
+                "slug": "touch-focus-bonus",
```
Comment (TikaelSol) [inline packs/data/feats.db/touch-focus.json:33]:
```suggestion
```

**Pull request merged**